tests/plugins/plugin.chapters.test.js

```diff
@@ -51,3 +100,38 @@ describe('Plugin: Menu Toggle', () => {
     expect($.ajax).toHaveBeenCalled();
   });
 });
+
+describe('updateTOCState', () => {
+
+  beforeEach(() => {
+    window.HTMLElement.prototype.scrollIntoView = function() {};
+    br.getPageIndex = (str) => parseInt(str);
```

Comment:
Use `sinon` here instead of manually stubbing this functions; that allows use to add an `afterEach` call with `sinon.restore()` and reset the environment back to normal.

```suggestion
    sinon.stub(window.HTMLElement.prototype, 'scrollIntoView');
    sinon.stub(br, 'getPageIndex').callsFake(parseFloat);
```

Reply:
Using sinon doesn't seem to work with scrollIntoView, even using Element as the object of the method